### PR TITLE
docs(interfaces): added documentation for Private Api Client interface

### LINKED ISSUE #156: Check and update if necessary the interfaces documentation : IPrivateApiClient (included) -> WsRpcMethods (included)

### PR DESCRIPTION
**Pull request by 0xSwapFeeder**
Aims to resolve #156.

### CODE REVIEW

**Comment by Elli610** [comment]
you forgot to run prettier

**Review by Elli610 — APPROVED** [review]

File: src/interfaces/IPrivateApiClient.ts [hunk 1]
```diff
@@ -1,15 +1,105 @@
...
+ * @export
+ *
+ */
 export interface IPrivateApiClient {
+  /**
+   * Method to ban a node by its ID.
+   *
+   * @param id - The ID of the node to ban.
+   *
+   * @returns - A promise so that the call can be asynchronous.
```
Comment (gregLibert) [inline src/interfaces/IPrivateApiClient.ts:16]: Is it normal that we don't knon if the action was successful or not ?

File: src/interfaces/IPrivateApiClient.ts [hunk 2]
```diff
@@ -1,15 +1,105 @@
 import { ISignedMessage } from './ISignedMessage';
 
+/**
+ * Private API client interface for creating an API to interact with a Massa Node.
+ *
+ * @interface IPrivateApiClient
```
Comment (gregLibert) [inline src/interfaces/IPrivateApiClient.ts:6]: Can you fixe Ci warning ?

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: src/interfaces/IPrivateApiClient.ts (hunk 1 above)
Comment (gregLibert) [reply src/interfaces/IPrivateApiClient.ts:12]: J'ai  fumé, désolé

**Review by gregLibert — APPROVED** [review]

**Pull request merged**